src/JuliusSweetland.OptiKey.Core/app.config

```diff
@@ -3,1424 +3,2363 @@
 Copyright (c) 2019 OPTIKEY LTD (UK company number 11854839) - All Rights Reserved
 -->

```

Comment:
Unless you have added/removed any application settings it's safest to remove these files changes from the PR completely.